avsomr/avsomr.cpp

```diff
@@ -0,0 +1,165 @@
+#include "avsomr.h"
+
+#include "avslog.h"
+
+using namespace Ms::avs;
```

Comment:
Should be `Ms::Avs`.

Reply:
You mean loosing the `;`?

Reply:
I mean capitalizing `a`.

Reply:
outch, need to have a word with my optician ;-)

Reply:
Seems to lead to a linker error on AppVeyor?